index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Portfolio Design" />
+    <meta name="author" content="Adeyemi Dada" />
+    <link href="assets/css/style.css" rel="stylesheet">
+    <title>Portfolio Design</title>
+</head>
+
+<body>
+    <header>
+        <ul class="menu">
+            <li class="logo"><a href="#" class="logo_link link text-center fz-18 fw-700 lh-20">My logo</a></li>
+            <li class="item"><a href="#">Portfolio</a></li>
+            <li class="item"><a href="#">About</a></li>
+            <li class="item"><a href="#">Contact</a></li>
+            <li class="mobile-menu"><a href="#">
+                    <img src="assets/images/header-menu.svg" alt="Hamburguer Icon">
+                </a></li>
+        </ul>
+    </header>
+    <section class="heading flex">
+        <h1 class="heading__title fw-700">I'm Adeyemi.<br /> Glad to see you!</h1>
+        <p class="heading__desc">
+            I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <p class="heading__connect">LET’S CONNECT</p>
+        <div class="heading__icons">
+            <ul class="heading__icons__ul ul flex">
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/twitter.svg" alt="twitter icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/linkedin.svg" alt="linkedin icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/medium.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/github.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/angellist.svg" alt="angellist icon"></a></li>
+            </ul>
+        </div>
+    </section>
```

Comment:
Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Make sure the attributes such as `font`, `colors`, `images`,` text`, `margins`, `background-image` etc are the same 😄 
The next Code reviewer will assess your work according to the update of your work, kindly consider the design in detail 

| Your design | Figma design |
| :---: | :---: |
|![Capture](https://user-images.githubusercontent.com/80458199/234919783-48c2fc57-9043-40cd-8831-d4abcbd02aa8.JPG) | ![Capture 2](https://user-images.githubusercontent.com/80458199/234919828-9283ee54-a4bd-471d-833e-184724de5254.JPG) |